docs/collect-user-data.md

```diff
@@ -0,0 +1,15 @@
+```
```

Comment:
Dumb question maybe, but if this data is contained in the actual script, is there also a reason to have an external markdown file as well?

Maybe drop this markdown file, or change the script to cat this file out? Just so they can't get out of sync.